src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -98,7 +97,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String date = (String) entity.getProperty("date");
 
       Election newElection =
-          Election.builder().setID(id).setName(name).setScope(scope).setDate(date).build();
+          Election.builder().setId(id).setName(name).setScope(scope).setDate(date).build();
```

Comment:
See my most recent PR -- we need to add the "propositions" and "candidates" sets in order to use the builder; I added empty HashSets for the time being.

Reply:
Will update!